build/sharpmake/src/base.cs

```diff
@@ -254,6 +254,11 @@ public override void Configure(RexConfiguration conf, RexTarget target)
     }
   }
 
+  protected string GetClangToolsPath(RexConfiguration conf)
+  {
+    return Path.Combine(conf.ProjectPath, "clang_tools", conf.Target.GetFragment<Compiler>().ToString(), conf.Name);
```

Comment:
Why does the file system API require the ClangToolsPath?

Reply:
it's some other boyscout changes that got into this change

Reply:
An you refrain from doing no related stuff on a feature branch?